cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -114,24 +115,56 @@
         {% else %}
             {% set corrections_and_notices_title = _("Corrections") %}
         {% endif %}
-        <div class="ons-container">
-            {% call onsDetails({"id": "corrections-and-notices", "title": corrections_and_notices_title}) %}
-                {% if page.corrections %}
-                    <h3>{{ _("Corrections") }}</h3>
-                    <div id="corrections">
-                        {% for correction in page.corrections %}
-                            {% include_block correction %}
-                        {% endfor %}
+        <details class="ons-corrections-notices ons-u-mb-l" open>
+            <summary class="ons-corrections-notices__banner ons-u-pt-s ons-u-pb-s">
+                <div class="ons-container ons-corrections-notices__banner-contents">
+                    <svg class="ons-corrections-notices__info-icon ons-u-mr-xs" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+                        <g id="Artwork_32x32" data-name="Artwork 32x32">
```

Comment:
I think we can remove the `<g>` wrapper as it only seems to contain meta information used by the image editor.

Reply:
I use https://jakearchibald.github.io/svgomg/ ;)

Reply:
Yeah fair point, I believe the SVG has been replaced by just styling/markup in the DS component but I'll remove this as it doesn't need to be there 👍

Reply:
Have replaced the svg with a styled icon instead